virttest/utils_disk.py

```diff
@@ -790,7 +790,7 @@ def get_disk_size(session, os_type, did):
         return get_disk_size_windows(session, did)
 
 
-def set_drive_letter(session, did, partition_no=1):
+def set_drive_letter(session, did, partition_no=1, target_letter=None):
```

Comment:
Please add docstring for parameter `target_letter`.Thanks.

Reply:
added it ,thanks